src/pages/regulations/View.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <template v-if="loaded">
+        <div class="flex items-center" v-if="regulation.summary">
+            <div class="text-lg font-medium">Özet </div>
+        </div>
+        <div class="mt-4" v-html="regulation.summary"></div>
+        <div class="p-5 mt-5 text-white intro-y box bg-primary">
+            <div class="flex items-center">
+                <div class="text-lg font-medium">Yönetmelik Metni</div>
+            </div>
+            <div class="mt-4" v-html="regulation.text"></div>
+        </div>
+        <div class="flex gap-3 m-5 justify-end">
+            <RouterLink :to="{ name: 'regulations.list' }">
+                <Button variant="primary" class="w-24 mb-2 mr-1">
+                    Geri
+                </Button>
+            </RouterLink>
+        </div>
+    </template>
+</template>
+
+<script setup lang="ts">
+import Button from "@/base-components/Button";
+import { useRegulationsStore } from "@/stores/regulations"
+import { onBeforeMount, ref } from "vue";
+import { useRoute } from "vue-router";
+const route = useRoute()
+const loaded = ref(false)
+const regulationsStore = useRegulationsStore()
+const regulation = ref({})
+onBeforeMount(async () => {
```

Comment:
konseptleri birbirinden ayırmak okunurluk artırmak için önemli 

örneğin değişken tanımlarıyla bu methodlar arası bi boşluk olsa
okurken aramıycaz nerede başlıyor nerede bitiyor diye